pkg/cmd/options/options_test.go

```diff
@@ -112,7 +114,9 @@ func TestApplyTo(t *testing.T) {
 			assert.NoErrorf(t, err, "Error while parsing flags")
 
 			serverConfig := genericapiserver.NewConfig(apiserver.Codecs)
-			err = o.ApplyTo(serverConfig)
+			clientConfig := &rest.Config{}
+			informers := informers.NewSharedInformerFactory(nil, 0)
+			err = o.ApplyTo(serverConfig, clientConfig, informers)
```

Comment:
This should most likely fix the failure I had in my other PR, thanks for looking into that.

The only thing that I see missing is that we never initialize the informer factory outside of the test. If we look at how the sample-apiserver is doing its initialization, we would need to add that in the `Config()` function: https://github.com/kubernetes/kubernetes/blob/15ee05afd4f9918cee0ab6dee459da3604957d85/staging/src/k8s.io/sample-apiserver/pkg/cmd/server/start.go#L125-L147

Reply:
On it, thanks!

Reply:
@dgrisonnet done, ptal if that's what was on your mind. Thanks!

Reply:
yes perfect :+1: